.github/workflows/Documenter.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@latest
         with:
-          version: '1.5'
+          version: '^1.6.0-rc1'
```

Comment:
```suggestion
          version: '1.6'
```
?

Reply:
GitHub actions will use v1.6 (since it's released), but that makes it easier for me to remember the correct syntax when updating for Julia v1.7 (in the RC stage).

Reply:
If you're sure about this, then it's OK for me. I'd add a comment, though, because someone else might stumble upon this (eg me, after forgetting about this conversation)